src/index.js

```diff
@@ -1,50 +1,76 @@
-import 'bootstrap/dist/css/bootstrap.min.css';
-import './style.css';
-import 'bootstrap';
-import 'bootstrap/js/dist/dropdown';
-import renderNav from './navbar';
-import renderHome from './home';
-import renderMenu from './menu';
-import renderContact from './contact';
-
-
-renderNav();
-renderHome();
-
-const tabLinks = document.querySelectorAll('.nav-link');
-const content = document.getElementById('content');
-
-const renderPage = (location, page) => {
-  location.innerHTML = '';
-  page();
-};
-
-const removeActive = () => {
-  tabLinks.forEach((link) => {
-    if (link.classList.contains('active')) {
-      link.classList.remove('active');
-    }
-  });
-};
-
-tabLinks.forEach((link) => {
-  link.addEventListener('click', (e) => {
-    if (e.target.textContent === 'Home') {
-      removeActive();
-      e.target.classList.add('active');
-      renderPage(content, renderHome);
-    }
-
-    if (e.target.textContent === 'Menu') {
-      removeActive();
-      e.target.classList.add('active');
-      renderPage(content, renderMenu);
-    }
-
-    if (e.target.textContent === 'Contact') {
-      removeActive();
-      e.target.classList.add('active');
-      renderPage(content, renderContact);
-    }
-  });
-});
\ No newline at end of file
+// Modules for each tab
+import { initialLoad, repeatedSetup } from './home'
+import { loadAbout } from './about'
+import { loadMenu } from './menu'
+import { loadContact } from './contact'
+import 'bootstrap'
+// CSS
+import './style.css'
+
+let content = document.getElementById('content');
+
+// Loading Home Contents
+content.appendChild(repeatedSetup());
+let main = content.querySelector('main');
+main.appendChild(initialLoad());
+
+
+/**
+ * Tab swtiching logic. If any tab is clicked, it will load the
+ * contents regarding to it
+ */
+let navBar = content.querySelector('ul');
+
+// Add eventListeners to all li
+for(let i = 0; i < navBar.children.length; i++) {
+  let li = navBar.children[i];
+
+  
+  li.addEventListener('click', (event) => {
+    let tab = li.innerHTML;
+
+    switch(tab) {
+      case 'Home':
+        main.innerHTML = '';
+        main.appendChild(initialLoad());
+
+        // Make a bottom line
+        navRemoveBottomLine();
+        li.style.borderBottom = "5px solid #20de83";        
```

Comment:
Instead of having your css inside your logic, you could use `classList` to add class then style it in your css files.
https://www.w3schools.com/jsref/prop_element_classlist.asp